src/fragments/lib/auth/js/overview.mdx

```diff
@@ -1,39 +1,30 @@
-	
-![Image](/images/SocialAuthZ.png)
-	
-You can also get credentials directly from Identity Pools by passing tokens from a provider directly to `Auth.federatedSignIn()`. However you will have to use that provider's SDK directly in your app and manage token refresh and auth flows manually.
-	
+
+Many apps also support login with social providers such as Facebook, Google Sign-In, or Login With Amazon. The preferred way to do this is via an OAuth redirect which lets users login using their social media account and a corresponding user is created in User Pools. With this design you do not need to include an SDK for the social provider in your app. Set this up by running `amplify add auth` and selecting the social provider option. Upon completion you can use `signInWithRedirect` in your application to either show a pre-built "Hosted UI" or pass in a provider name (e.g. `signInWithRedirect({provider: 'Facebook'})`) to interface directly and build out your own UI. ![Image](/images/SocialAuthZ.png) You can also get credentials directly from Identity Pools by passing tokens from a provider directly to `signInWithRedirect`. However you will have to use that provider's SDK directly in your app and manage token refresh and auth flows manually.
 
 ## Accessing AWS services
 
-Some apps need to use AWS services which require [signing requests](https://docs.aws.amazon.com/general/latest/gr/signing_aws_api_requests.html). Examples of this would be storing images or videos on S3, or sending analytics to Pinpoint or Kinesis. Amplify automatically signs requests with short term credentials from a Cognito Identity Pool which automatically expire, rotate, and refresh by the Amplify client libraries. Setting up your backend with `amplify add auth` and calling `Auth.signIn` will automatically do this for you as well after the client authenticates. The diagram below shows how JWT tokens are returned from User Pools and AWS credentials from Identity Pools. You can access these at any time with `Auth.currentSession()` and `Auth.currentCredentials()`.
-	
-![Image](/images/AWSAuthZ.png)
+Some apps need to use AWS services which require [signing requests](https://docs.aws.amazon.com/general/latest/gr/signing_aws_api_requests.html). Examples of this would be storing images or videos on S3, or sending analytics to Pinpoint or Kinesis. Amplify automatically signs requests with short term credentials from a Cognito Identity Pool which automatically expire, rotate, and refresh by the Amplify client libraries. Setting up your backend with `amplify add auth` and calling `signIn` will automatically do this for you as well after the client authenticates. The diagram below shows how JWT tokens are returned from User Pools and AWS credentials from Identity Pools. You can access these at any time with `fetchAuthSession`. ![Image](/images/AWSAuthZ.png)
```

Comment:
```suggestion
Some apps need to use AWS services which require [signing requests](https://docs.aws.amazon.com/general/latest/gr/signing_aws_api_requests.html). Examples of this would be storing images or videos on S3, or sending analytics to Pinpoint or Kinesis. Amplify automatically signs requests with short term credentials from a Cognito Identity Pool which automatically expire, rotate, and refresh by the Amplify client libraries. Setting up your backend with `amplify add auth` and calling `signIn` will automatically do this for you as well after the client authenticates. The diagram below shows how JWT tokens are returned from User Pools and AWS credentials from Identity Pools. You can access these at any time with `fetchAuthSession`.

![Image](/images/AWSAuthZ.png)
```